wagtail/wagtailadmin/static_src/wagtailadmin/js/core.js

```diff
@@ -52,6 +52,9 @@ function initTagField(id, autocompleteUrl) {
  *    - alwaysDirty - When set to true the form will always be considered dirty,
  *      prompting the user even when nothing has been changed.
 */
+
+var canGiveWarning = 1;
```

Comment:
Could this be renamed to something like ``disableDirtyFormCheck``? (I think ``canGiveWarning`` is a bit to generic)